### PR TITLE
fix (abr_testing): Fix for make simulate command

### PR DESCRIPTION
**Pull request by AnthonyNASC20**
# Overview
Fixes automation process for simulating protocols contained in the abr_testing folder

### CODE REVIEW

File: abr-testing/abr_testing/protocol_simulation/abr_sim_check.py [hunk 1]
```diff
@@ -1,34 +1,109 @@
...
     # Directory to search
+    global root_dir
     root_dir = "abr_testing/protocols"
-
+    global exclude
     exclude = [
         "__init__.py",
+        "helpers.py",
         "shared_vars_and_funcs.py",
+        "9_parameters.csv",
```
Comment (rclarke0) [inline abr-testing/abr_testing/protocol_simulation/abr_sim_check.py]: Why is this being excluded?

File: abr-testing/abr_testing/protocol_simulation/simulation_metrics.py [hunk 2]
```diff
@@ -241,6 +266,8 @@ def parse_results_volume(
         "Right Pipette Total Aspirates",
         "Right Pipette Total Dispenses",
         "Gripper Pick Ups",
+        "Gripper Pick Ups of opentrons_tough_pcr_auto_sealing_lid",
```
Comment (rclarke0) [inline abr-testing/abr_testing/protocol_simulation/simulation_metrics.py]: this should not be repeated twice

File: abr-testing/abr_testing/protocol_simulation/simulation_metrics.py [hunk 3]
```diff
@@ -302,6 +329,8 @@ def parse_results_volume(
     total_time_row.append(str(end_time - start_time))
 
     for metric in metrics:
+        print(f"Dictionary: {metric}\n\n")
+        print(f"Dictionary: {metric}\n\n")
```
Comment (rclarke0) [inline abr-testing/abr_testing/protocol_simulation/simulation_metrics.py]: repeat

File: abr-testing/abr_testing/protocol_simulation/simulation_metrics.py [hunk 4]
```diff
@@ -445,7 +529,8 @@ def main(
             choice = ""
             while not choice:
                 choice = input(
-                    "Remove air_gap commands to ensure accurate results? (Y/N): "
+                    "Remove air_gap commands to ensure accurate results: (continue)? (Y/N): "
+                    "Remove air_gap commands to ensure accurate results: (continue)? (Y/N): "
```
Comment (rclarke0) [inline abr-testing/abr_testing/protocol_simulation/simulation_metrics.py]: this was repeated twice. also why is "continue: added

File: abr-testing/abr_testing/protocol_simulation/simulation_metrics.py [hunk 5]
```diff
@@ -445,7 +529,8 @@ def main(
             choice = ""
             while not choice:
                 choice = input(
-                    "Remove air_gap commands to ensure accurate results? (Y/N): "
+                    "Remove air_gap commands to ensure accurate results: (continue)? (Y/N): "
```
Comment (sfoster1) [inline abr-testing/abr_testing/protocol_simulation/simulation_metrics.py]: fun fact: soon you will not have to do this

**Review by sfoster1 — APPROVED** [review]
Nice!

**Review by rclarke0 — APPROVED** [review]

**Pull request merged**